fuel/src/main/kotlin/com/github/kittinunf/fuel/core/requests/DefaultRequest.kt

```diff
@@ -392,45 +406,62 @@ data class DefaultRequest(
     }
 
     override fun response(handler: ResponseResultHandler<ByteArray>) =
-        response(ByteArrayDeserializer(), handler)
+            response(ByteArrayDeserializer(), handler)
```

Comment:
You probably didn't want to change this indentation to 8 instead of 4.

Here and below.